### ISSUE TITLE
Feature Request - Allow for Configuration of LineTextLabel and CallDeclineAll for Primary Line

### ISSUE THREAD

**Issue by MikeRalston98** [kctrey/wxcadm#7]
Would love to see the device class improved with the ability to set LineTextLabel and CallDeclineAll for the primary line of a device. Currently the "add" method is available for adding secondary/shared lines to an existing device, and for those you can set these parameters but the existing lines are automatically added to the list so they cannot be changed. If this functionality could be added into the "create" type functions for devices as well, that would be amazing. Currently creating a device looks something like the following:

person.add_device(model=model, mac=macaddress)

It would be great to be able to do something similar to the following and set the line configuration as the device is created:

person.add_device(model=model, mac=macaddress, linelabel='User Label XYZ', allowCallDeclineEnabled=False)

Thanks for all your hard work on this repo. Extremely helpful when interacting with the Webex Calling APIs.

**Comment by kctrey** [comment]
@MikeRalston98 The Person.add_device() method can't be used to add a Person to an existing Device, so what you might be looking for is the Device.members.add() method, which does have those fields available. I do have an enhancement planned for the next release that allows you to modify existing DeviceMember entries to change those values, so, if you are creating a new device with Person.add_device(), you would be able to use the returned Device instance to modify the members.

What I will probably do is add a method in the DeviceMemberList to allow you to find the member by the Person, making that easier. What I envision is something like the following:

```
new_device = person.add_device(model=model, mac=mac)
member = new_device.members.get(person=person)
member.set_line_label('New Line Label')
member.set_allow_call_decline_enabled=False
```

or maybe a shorted method that allows both in one call. Does that make sense and would it fit your needs? That method aligns better to the Webex logic where the Person/Workspace is a "member" of the device rather than being an "owner" of the Device.

**Comment by kctrey** [comment]
I realized that your first paragraph already mentioned what I pointed out. Regardless, I think the way I'll have to do it is to just allow you to set it at the DeviceMember level. Plus that will give more flexibility for any future changes.

**Comment by MikeRalston98** [comment]
@kctrey Yes, that would be exactly what I'm looking for. Building out a script for migrations from Call Manager and after I assign the user licenses, I add the device with person.add_device. Often during the migrations, they'll want the line label set to what they had before and also want the ability to have calls decline only on that device since they'll have shared lines and whatnot. 

The methods you posted in your reply would be perfect. I personally don't see a need for an all-in-one type call for those types of settings. Appreciate the quick response.

**Comment by kctrey** [comment]
@MikeRalston98 I had to change direction a little, because I forgot what I return from the Person.add_device(). Because I return a dict, you'll have to do the following (or similar):

``` python
device_details = person.add_device(model=model, mac=macaddress)
device = device_details['device_object']
member = device.members.get(person=person)
member.set_line_label('New Label')
member.set_call_decline_all(True)  # or False if that's what you want
```

The changes are in the `dev` branch now. All you'd have to do is replace `device.py` with the `device.py` from the dev branch to test. I have been testing and it seems to work well, but would be happy to have another tester.

**Comment by kctrey** [comment]
This is fixed in 4.2.6
